pkg_for_ros/personalInfo PersonalData
string Job_DoneORNot

